f8a_worker/storages/s3_userintent.py

```diff
@@ -9,3 +9,16 @@ def store_in_bucket(self, input_json):
             existing_intent_list = existing_object.get('intent', [])
             input_json.get('intent', []).extend(existing_intent_list)
         self.store_dict(input_json, key)
+
+    def store_master_tags(self, input_json):
+        if 'ecosystem' not in input_json or 'data' not in input_json:
+            raise ValueError("Ecosystem and data are needed in supplied JSON, got "
+                             "{} instead".format(input_json))
+        self.store_dict(input_json['data'], input_json['ecosystem'] + '/manual_tag_list.json')
+
+    def fetch_master_tags(self, ecosystem):
+        if not ecosystem:
+            raise ValueError("Ecosystem is needed to fetch the master tag list")
+        file_name = ecosystem + '/manual_tag_list.json'
```

Comment:
`file_name = os.path.join(ecosystem, 'manual_tag_list.json')`

Reply:
thanks. let me fix this.

Reply:
this is not the actual path. this is just a prefix which acts as a path for the json file in the bucket

Reply:
ok 👍